### PR TITLE
[WIP] Changing org space to location and making visual changes to org selector

### PR DESCRIPTION
**Pull request by adamjgrant**
https://app.asana.com/0/45834325347620/46149353682004

### CODE REVIEW

**Comment by adamjgrant** [comment]
screen shot 2015-09-04 at 3 26 25 pm

@andrewconner @cvializ

**Comment by adamjgrant** [comment]
Damnit. Forgot a few things.
- Still need the banner text describing what the privacy setting means.
- "Members of [org name] can join, add, and remove keeps" checkbox.

**Comment by cvializ** [comment]
Is that the same gold as the extension? It looks different to me, and I think they ought to be the same

**Comment by adamjgrant** [comment]
@cvializ Probably not. I can't update my extension for some reason. Can you give me a hex to match?

**Comment by cvializ** [comment]
It's the `@vis-color` variables below

``` less
.kifi-keep-box-lock-icon {
  @vis-color: #d81615;
  @lock-url: %(@svgLibCardLockDataUrl, #000);
  @lock-selected-url: %(@svgLibCardLockDataUrl, #d81615);
  width: 15px;
  background: url(@lock-url) center no-repeat;

  :checked ~ & {
    background: url(@lock-selected-url) center no-repeat;

    & + span {
      color: darken(@vis-color, 10%);
    }
  }
}
.kifi-keep-box-org-icon {
  @vis-color: #daa520;
  @org-url: %(@svgLibCardOrgPrivateDataUrl, #000);
  @org-selected-url: %(@svgLibCardOrgPrivateDataUrl, #daa520);
  background: url(@org-url) center no-repeat;

  :checked ~ & {
    background: url(@org-selected-url) center no-repeat;

    & + span {
      color: darken(@vis-color, 10%);
    }
  }
}
.kifi-keep-box-globe-icon {
  @vis-color: #56b263;
  @globe-url: %(@svgLibCardGlobeDataUrl, #000);
  @globe-selected-url: %(@svgLibCardGlobeDataUrl, #56b263);
  background: url(@globe-url) center no-repeat;

  :checked ~ & {
    background: url(@globe-selected-url) center no-repeat;

    & + span {
      color: darken(@vis-color, 10%);
    }
  }
}
```

**Comment by adamjgrant** [comment]
screen shot 2015-09-04 at 4 22 46 pm

good now @andrewconner @cvializ

File: kifi-backend/modules/shoebox/angular/src/common/components/grid.styl [hunk 1]
```diff
@@ -4,9 +4,9 @@
 .kf-main-pane
   column(8)
 
+.kf-triptych
 .kf-side-pane
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/common/components/grid.styl:8]: you're making the codebase 110% classier

**Comment by cvializ** [comment]
:+1: :+1:

File: kifi-backend/modules/shoebox/angular/src/common/components/grid.styl (hunk 1 above)
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/common/components/grid.styl:8]: Can you see the brandy stains on the code?

**Pull request merged**